decentraland/voting
# #32 Docker compose

# PR 

## What does this PR do?
- Creates a compose network to orchestrate the 3 server
## How can we test it?
- Clone the repo
- Ensure you have docker daemon installed (mac)
- Run 
```bash 
docker-compose build
docker-compose up
```
Web should be visible through port `8080`
## Notes:
It's still missing a container with `nginx` to make the dev experience flawless. The webapp is still miss configure when trying to connect to web3, @nachomazzara take a look a this please.
